hadoop-ozone/common/src/main/proto/OzoneManagerProtocol.proto

```diff
@@ -471,12 +500,55 @@ message OzoneAclInfo {
     repeated OzoneAclRights rights = 3;
 }
 
+message GetAclRequest {
+  required OzoneObj obj = 1;
+}
+
+message GetAclResponse {
+  repeated OzoneAclInfo acls = 1;
+}
+
+message AddAclRequest {
+  required OzoneObj obj = 1;
+  required OzoneAclInfo acl = 2;
+}
+
+message AddAclResponse {
+  required bool response = 1;
+}
+
+message RemoveAclRequest {
+  required OzoneObj obj = 1;
+  required OzoneAclInfo acl = 2;
+}
+
+message RemoveAclResponse {
+  required bool response = 1;
+}
+
+message SetAclRequest {
+  required OzoneObj obj = 1;
+  repeated OzoneAclInfo acl = 2;
+}
+
+message SetAclResponse {
+  required bool response = 1;
+}
+
+message DeleteAclRequest {
```

Comment:
What's the difference between RemoveAclRequest and DeleteAclRequest? I don't see the later being used.

Reply:
what is  CLIENT_IP = 5; ? Not part of this change.

Reply:
Looks redundant, removed.

Reply:
@anuengineer CLIENT_IP was discussed when we added Acl api for Ranger. Idea is to support acls based on ip/ip-range. Might be of use in future.

Reply:
ok